src/fides/api/ops/models/messaging.py

```diff
@@ -97,14 +96,12 @@ class MessagingConfig(Base):
     )  # Type bytea in the db
 
     @classmethod
-    def get_configuration(cls, db: Session, service_type: str) -> Base:
+    def get_configuration(cls, db: Session, service_type: str) -> OrmWrappedFidesBase:
         """
         Fetches the configured MessagingConfig record by service type. Once fetched this function validates that
         the MessagingConfig is configured with secrets.
         """
-        instance: Optional[Base] = cls.get_by(
-            db=db, field="service_type", value=service_type
-        )
+        instance = cls.get_by(db=db, field="service_type", value=service_type)
```

Comment:
so base_class seems to define `declarative_base(cls=OrmWrappedFidesBase)` what is prompting this OrmWrappedFidesBase usage?